go/kubebuilder/memcached-operator/README.md

```diff
@@ -41,16 +41,18 @@ Build the Memcached operator image and push it to a public registry, such as qua
 ```
 $ make install
 $ export IMG=quay.io/example-inc/memcached-operator:v0.0.1
-$ make docker-build $IMG
-$ docker push $IMG
-$ make deploy $IMG
+$ make docker-build IMG=$IMG
+$ docker push IMG=$IMG
 ```
 
 **NOTE** The `quay.io/example-inc/memcached-operator:v0.0.1` is an example. You should build and push the image for your repository.
+As this example showcases validation webhook creation, please follow [this][certmanager] guide to install cert-mamager into cluster prior to deployment.
 
 Please verify expected result.
 
 ```shell
+$ make deploy IMG=$IMG
```

Comment:
make deploy is the command that do the deploy and in this way, IMO should be inside of the section that is missing here `Deploying your project in the Cluster`. After performing, the deploy than we can check the results.